packages/experience-builder-sdk/src/communication/MouseOverHandler.ts

```diff
@@ -5,24 +5,36 @@ import { sendMessage } from './sendMessage';
 export class MouseOverHandler {
   private currentHoveredElementId: string | null = null;
 
+  private getMargins = (element: HTMLElement) => {
+    const styles = window.getComputedStyle(element);
```

Comment:
Can this code be executed on server? If so, please add the undefined check. Otherwise, it can break on SSR/SSG

Reply:
That's a valid point 👍 I will add this